ci-operator/step-registry/openshift-qe/rosa-loaded-upgrade/openshift-qe-rosa-loaded-upgrade-workflow.yaml

```diff
@@ -0,0 +1,8 @@
+
+workflow:
+  as: openshift-qe-rosa-loaded-upgrade
+  steps:
+    test:
+      - ref: openshift-qe-rosa-loaded-upgrade
```

Comment:
Same comment as below, think this can't be used so we should delete it

Reply:
+1 we will not be able to use openshift-qe-rosa-loaded-upgrade workflow

Reply:
Update in new code, thanks!